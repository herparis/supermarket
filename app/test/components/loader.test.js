import React from 'react'
import { shallow } from 'enzyme'

import Loader from '../../src/components/commons/loader/loader'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('Loader components rendeding', () => {
  const loader = shallow(<Loader />)
  it('loader renders', () => {
    checkRender(loader)
  })
})
