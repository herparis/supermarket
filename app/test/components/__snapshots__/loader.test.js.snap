// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader components rendeding loader renders 1`] = `ShallowWrapper {}`;
